server/Authentication/authentication.ts

```diff
@@ -0,0 +1,45 @@
+import express from 'express'
+import dotenv from 'dotenv';
+import bcrypt from 'bcrypt';
+import { validateUserCredentials } from '../middlewares/registrationMiddleware'
+import client from '../database/database'
+
+const app = express();
+app.use(express.json());
+dotenv.config();
+
+//Test porpouses only
+app.get('/registration', (req: express.Request, res: express.Response) => {
+    const test = client.query('SELECT * FROM users');
+    console.log(test);
+    res.status(200).send(test)
+})
+
+app.post('/registration', validateUserCredentials, async (req: express.Request, res: express.Response) => {
+    const { firstname, lastname, email, username, password, dayOfBirth, monthOfBirth, yearOfBirth } = req.body;
+
+    //Creating new variable in order to hash the password
+    let hashedPassword: string = password;
```

Comment:
why create two variables for the same thing? one is enough

Reply:
Done